packages/amplication-build-manager-service/.env

```diff
@@ -6,6 +6,7 @@ GENERATE_RESOURCE_TOPIC=build.internal.generate-resource.event.0
 BUILD_STATUS_TOPIC=build.internal.build-status.event.0
 
 BUILD_CONTEXT_FS_LOCATION=/home/eugene/build-contexts
```

Comment:
Make sure to remove `/eugene` mentions in the `.env` file

Reply:
Done